soccer/src/soccer/strategy/agent/position/penalty_player.hpp

```diff
@@ -40,7 +40,7 @@ class PenaltyPlayer : public Position {
 private:
     std::optional<RobotIntent> derived_get_task(RobotIntent intent) override;
 
-    enum State { LINE_UP, SHOOTING_START, SHOOTING };
+    enum State { LINE_UP, SHOOTING_START, SHOOTING, SMALL_KICK, LINE_UP_2 };
```

Comment:
I think we can come up with a more descriptive name than LINE_UP_2. Feel free to change LINE_UP as well